minecraft/protocol/packet/event.go

```diff
@@ -118,6 +118,12 @@ func (pk *Event) Unmarshal(r *protocol.Reader) {
 		pk.EventData = &protocol.MovementCorrectedEventData{}
 	case EventTypeExtractHoney:
 		pk.EventData = &protocol.ExtractHoneyEventData{}
+	case EventTypePlayerWaxedOrUnwaxedCopper:
+		pk.EventData = &protocol.WaxedOrUnwaxedCopperEventData{}
+	case EventTypeSneakCloseToSculkSensor:
+		pk.EventData = &protocol.SneakCloseToSculkSensorEventData{}
+	default:
+		pk.EventData = &protocol.UnknownEventData{}
```

Comment:
I would rather still panic here (it should be picked up by the `defer recover()` later on) so that we see errors when they occur.